packages/node_modules/samples/browser-socket/app.js

```diff
@@ -69,17 +69,39 @@ document.getElementById('credentials').addEventListener('submit', (event) => {
       webex.messages.listen()
         .then(() => {
           console.log('connected');
-          console.log('listening to messages');
+          console.log('listening to message events');
           updateStatus(true);
           webex.messages.on('created', (message) => {
-            console.log('created');
+            console.log('message created event:');
             console.log(message);
           });
         })
         .catch((err) => {
           console.error(`error listening to messages: ${err}`);
           updateStatus(false);
         });
+      webex.memberships.listen()
+        .then(() => {
+          console.log('connected');
```

Comment:
Is the intent to replace these logs with assertions at some point?

Reply:
I was just scaffolding of your original sample.   I think we ultimately need a better sample